app/controllers/hackathons/submissions_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
     end
 
     if @hackathon.save context: :submit
+      NotifyAdmins.call(hackathon_title: @hackathon.name)
```

Comment:
Rather than having this in the controller, I would suggest having this at a lower level in the domain model, especially since it's more on-the-side - when you're submitting a hackathon, the core "essence" of that isn't emailing admins. [I don't see a great reason for us to start using service objects for this](https://www.codewithjason.com/code-without-service-objects), but maybe we could make a `Hackathon` [`Deliverable`](https://github.com/hackclub/hackathons-backend/blob/2b50b94fcd440dde98f0ff9764fca5c67876e6d8/app/models/concerns/deliverable.rb).

Reply:
I see. Once again, not familiar with concerns, but I will look into it (both concerns concept and this blog post about not using service objects). Thanks for your reply!